x/oracle/keeper/keeper_test.go

```diff
@@ -20,24 +20,24 @@ const (
 )
 
 func TestCreateGetProphecy(t *testing.T) {
```

Comment:
This test is failing

Reply:
find the root cause, the validators' order from test helper not the same with predefined.